audio/decoders/wma.cpp

```diff
@@ -26,7 +26,7 @@
 #include "common/math.h"
 #include "common/error.h"
 #include "common/memstream.h"
-#include "common/compression/huffman.h"
+#include "common/huffman.h"
```

Comment:
What's wrong with huffman? It is a compression algorithm

Reply:
True, but it doesn't really fit in with the other files in that directory because it's a building block for implementing compression methods rather than one in and of itself, so it felt like it would be a better fit keeping it next to files like `bitstream.h`.

Reply:
Then I'd suggest moving bitstream.h to compression too :D